docs/source/developer/contributing.rst

```diff
@@ -251,6 +251,17 @@ pass without a reply that unblocks it.
 Our expectation is that every new issue should be examined within a week of
 its creation.
 
+Triagers should label easier issues as ``good first issue`` to
```

Comment:
Suggest replacing "easier" with a less ambiguous term such as low effort/low complexity

Reply:
That's a bit tricky. I think we should not say that good first issues are low effort - this can be demotivating especially if it ends up being a lot of effort for a beginner; they certainly should not be high complexity issues as there is a compelxity in learning a new codebase and contribution process itself. It might be hard to put this into words. I will use "easier/lower complexity" for now, please suggest a better wording if something comes to mind.

For example, GitHub uses "_selected for its relative approachability_" (https://github.com/jupyterlab/jupyterlab/contribute):

> Each issue displayed here is a "good first issue," selected for its relative approachability for first-time contributors.

That's a mouthful, but maybe good?